### ISSUE TITLE
python version coredump

### ISSUE THREAD

**Issue by daohu527** [wheelos/core#41]
python version coredump

```
source scripts/env/runtime.bash
./bazel-bin/cyber/python/cyber_py3/examples/listener

# another terminal
source scripts/env/runtime.bash
./bazel-bin/cyber/python/cyber_py3/examples/talker
```

## error message
```
E20240725 01:30:21.466540 138203629369152 py_cyber.cc:94] PyObjectToPtr failed,type->apollo_cyber_pynodepyobj: 0x7db200000000
E20240725 01:30:21.466644 138203629369152 py_cyber.cc:610] cyber_PyNode_register_message:node ptr is null! desc->
cyber/proto/unit_test.protoapollo.cyber.proto"1
UnitTest

class_name (	
	case_name (	"S
Chatter
	timestamp (
lidar_timestamp (
seq (
content ("?
ChatterBenchmark
stamp (
seq (
content (	
```

**Comment by daohu527** [comment]
https://github.com/wheelos/core/issues/23

**Comment by daohu527** [comment]
fixed by #44
